src/commands/run.ts

```diff
@@ -109,16 +92,18 @@ async function runFunction(functionName: string, params: string[], scriptId: str
           // The 401 is probably due to this error:
           // "Error: Local client credentials unauthenticated. Check scopes/authorization.""
           // This is probably due to the OAuth client not having authorized scopes.
-          console.log(`` +
-            `Hey! It looks like you aren't authenticated for the scopes required by this script.
+          console.log(
+            `` +
+              `Hey! It looks like you aren't authenticated for the scopes required by this script.
```

Comment:
Weird spacing.

Reply:
`npm run prettier` (at least as it is configured now) format it like this

Changing how the string is created will change how it goes "prettier" probably

Reply:
Changed it slightly and it is now "normal"